packages/core/src/render3/instructions/element.ts

```diff
@@ -39,7 +43,7 @@ function elementStartFirstCreatePass(
 
   const hasDirectives =
       resolveDirectives(tView, lView, tNode, getConstant<string[]>(tViewConsts, localRefsIndex));
-  ngDevMode && logUnknownElementError(tView, native, tNode, hasDirectives);
+  ngDevMode && logOrThrowUnknownElementError(tView, native, tNode, hasDirectives);
```

Comment:
Let's may be rename this function as well?
```suggestion
  ngDevMode && validateElementIsKnown(tView, native, tNode, hasDirectives);
```

Reply:
It'd be also great to add some docs to that function to describe the goal of that function.

Reply:
While documenting it, I also spotted that this could be simplified (tView is not needed, just the schemas, and tNode neither, just the tag name).

I opened #45492 to avoid too much noise here, and I'll rebase this one when it lands.